Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1149,6 +1149,11 @@ bool NetPlayClient::DecompressPacketIntoFile(sf::Packet& packet, const std::stri
     return true;
 
   File::IOFile file(file_path, "wb");
+  if (!file)
+  {
+    PanicAlertT("Failed to open file. Verify your write permissions.");
```

Comment:
This should likely indicate the file. e.g.

```cpp
PanicAlertT("Netplay client failed to open file \"%s\". Verify your write permissions.", file_path.c_str());
```

so that a user knows which file they need to check (or find the directory, etc). An error message that just says  an equivalent of "Something bad happened" without any clarifying info doesn't give them much to go on to try and solve the issue.